song_translator/track/urls.py

```diff
@@ -1,13 +1,19 @@
-from django.urls import path
+from django.urls import path, re_path
 from rest_framework.urlpatterns import format_suffix_patterns
 
-from .views import TrackList, TrackDetail, SingerList, SingerDetail, TrackTranslate
-
+from .views import TrackList, TrackDetail, SingerList, SingerDetail, track_translate
 
+lang = ['af', 'sq', 'am', 'ar', 'hy', 'az', 'eu', 'be', 'bn', 'bs', 'bg', 'ca', 'ceb', 'ny', 'co',
```

Comment:
could you define your languages in one place? currently it's also defined in utils